packages/plugins/ui-theme/src/pages/home/Home.tsx

```diff
@@ -1,36 +1,20 @@
-import React, { useEffect, useState, useCallback } from 'react';
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
 
 import Loading from 'verdaccio-ui/components/Loading';
-import { useAPI } from 'verdaccio-ui/providers/API/APIProvider';
 
-import { PackageList } from './PackageList';
+import { Dispatch, RootState } from '../../store/store';
 
-interface Props {
-  isUserLoggedIn: boolean;
-}
+import { PackageList } from './PackageList';
 
-const Home: React.FC<Props> = ({ isUserLoggedIn }) => {
-  const [packages, setPackages] = useState([]);
-  const { getPackages } = useAPI();
-  const [isLoading, setIsLoading] = useState(true);
-  const loadPackages = useCallback(async () => {
-    try {
-      const packages = await getPackages();
-      // FIXME add correct type for package
-      setPackages(packages as never[]);
-    } catch (error: any) {
-      // FIXME: add dialog
-      // eslint-disable-next-line no-console
-      console.error({
-        title: 'Warning',
-        message: `Unable to load package list: ${error.message}`,
-      });
-    }
-    setIsLoading(false);
-  }, [getPackages]);
+const Home: React.FC = () => {
+  const packageStore = useSelector((state: RootState) => state.packages);
```

Comment:
If you only use .response,  it's much better to use the selector, because if `response` doesn't change won't re-render

Reply:
Thanks :) nice feedback